chunking_methods/gen_csv_labels.py

```diff
@@ -0,0 +1,177 @@
+"""Generates binary annotations using TweetyNet from weakly labeled audio.
+This file should be run from inside the PyHa directory. It also requires
+WTS_chunking.py to be added to the PyHa directory. 
+Input:     A path to a folder with audio files
+Output:    A csv with chunked, strongly-labeled annotations
+"""
+
+import os
+import sys
+from math import ceil
+import pandas as pd
+from pydub import AudioSegment
+# pylint: disable=import-error #this file gets put into PyHa
+from PyHa.IsoAutio import generate_automated_labels
+from WTS_chunking import dynamic_yan_chunking
+
+ISOLATION_PARAMETERS = {
+    "model" : "tweetynet",
+    "tweety_output": True,
+    "verbose" : True
+}
+
+CHUNK_DURATION = 5
+# incoming filetype
+FILETYPE = ".mp3"
+# use sliding window or raw chunks
+SLIDING_CHUNKS = False
+
+# input weak annotations
+METADATA_PATH = "~/metadata.csv"
+# output for strong annotations
+STRONG_LABELS_CSV = "~/132PeruXC_Labels.csv"
+# output for chunked annotations
+CHUNKED_CSV = "~/132PeruXC_RawChunks.csv"
+
+def convert2wav(path):
```

Comment:
No error message for bad extension, recommend creating a runtime error for this instead of silent return